app/main.py

```diff
@@ -573,6 +578,16 @@ def serve(self):
                         (env.repo, env.charset), get_vars)
                 response.out.write(content)
 
+        if config.get('enable_react_ui'):
+            if env.repo == 'static':
+                self.serve_static_content(self.env.action)
```

Comment:
Optional: Static contents are served as /global/static/... in the existing app. Maybe the new UI can do the same so that it doesn't require this special handling? Can be done later though.

Reply:
Done in PR #495.

Reply:
Added an explanation to the readme in PR #495; there's a TODO here to make it /global/static.